cmd/down.go

```diff
@@ -83,11 +91,27 @@ func Down() *cobra.Command {
 
 	cmd.Flags().StringVarP(&devPath, "file", "f", utils.DefaultManifest, "path to the manifest file")
 	cmd.Flags().BoolVarP(&rm, "volumes", "v", false, "remove persistent volume")
+	cmd.Flags().BoolVarP(&all, "all", "A", false, "deactivate all running dev containers")
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "namespace where the down command is executed")
 	cmd.Flags().StringVarP(&k8sContext, "context", "c", "", "context where the down command is executed")
 	return cmd
 }
 
+func allDown(ctx context.Context, manifest *model.Manifest, rm bool) error {
+	if len(manifest.Dev) == 0 {
+		return fmt.Errorf("okteto manifest has no 'dev' section. Configure it with 'okteto init'")
+	}
+	for _, dev := range manifest.Dev {
+		if err := runDown(ctx, dev, rm); err != nil {
+			analytics.TrackDown(false)
```

Comment:
Should we track the down event on all the defined devs? 
What happens if the dev environment is not on dev mode? We don't fail but we are tracking that event

Reply:
What happens now if the service is not on dev mode and the user runs `okteto down <service>` ?

Reply:
I'm not sure what `TrackDown` does, lmk what you decide and I can update the PR accordingly then.